public/modules/base/services/Role.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+/* Services */
+angular.module('mean').service('roleService', function ($http) {

```

Comment:
Idea/suggestion: consider using resources to encapsulate the boilerplate for RESTful access.

https://docs.angularjs.org/api/ngResource/service/$resource